src/renderer/components/dialogs/ContactDetail.js

```diff
@@ -0,0 +1,33 @@
+const React = require('react')
+
+const {
+  Classes,
+  Dialog
+} = require('@blueprintjs/core')
+
+class ContactDetailDialog extends React.Component {
+  render () {
+    var { contactId, onClose } = this.props
+    var isOpen = !!contactId
```

Comment:
Why not to use `const`?

Reply:
@alfaslash No reason :) @karissa been hacking hard and in the flow